change/@ni-nimble-components-f4776f02-4ee0-4534-8a00-bbc51b09d9c3.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I haven't taken the time to fully understand the issue and its history, but why are we only focused on buttons? Would the same issue affect other components? If so, is adding that test coverage considered part of #503?

Reply:
It was only with buttons that an alignment issue came up. [When adding icon support to the button](https://github.com/ni/nimble/pull/124), an alignment issue was discovered and worked around. It would make sense to add tests for other controls if we had some suspicion/history of them having a similar alignment issue. The only other control that supports icons and can be laid out in a row is the text field, and I verified that it doesn't have an alignment issue.